ports/stm32/mpconfigport.h

```diff
@@ -206,35 +206,6 @@ extern const struct _mp_obj_module_t mp_module_onewire;
 #define MACHINE_BUILTIN_MODULE_CONSTANTS
 #endif
 
-#if MICROPY_PY_UTIME
```

Comment:
there are a bunch of `extern const struct _mp_obj_module_t ...` above that can now be deleted

Reply:
Done.